README.md

```diff
@@ -1,295 +1,202 @@
+  be overwritten**, so be careful!
 
 ```bash
-$ mkdir name_of_directory
+$ cp hello_world.rb hello_world_again.rb name_of_directory
 ```
-Now if you enter `ls` you should see the empty directory you just created in the working
-directory.
+Now you have too many files, or some of them are no longer needed. Let's talk
+about how to remove files.
 
 ### Removing Files With `rm`
 
-To delete a file, we can enter `rm` at a shell prompt.
-**Note:** Deleting a file with rm is *permanent*. This action cannot be undone.
+To delete a file, we can enter `rm` at a shell prompt. This command stands for
+**r**e**m**ove.
 
 ```bash
 $ rm hello_world.rb
 ```
+**Note:** Deleting a file with rm is *permanent*. This action cannot be undone.
 
 There additional options to `rm`:
 
-* -i (interactive) — Prompts you to confirm the deletion. This option can stop you from
-deleting a file by mistake.
-* -f (force) — Overrides interactive mode and removes the file(s) without prompting.
-Use this with caution. This action cannot be undone!
+* -i (interactive) — Prompts you to confirm the deletion. This option can stop
+  you from deleting a file by mistake.
+* -f (force) — Overrides interactive mode and removes the file(s) without
+  prompting. Use this with caution. This action cannot be undone!
 * -v (verbose) — Shows the progress of the files as they are being removed.
-* -r (recursive) — Deletes a directory and all files and subdirectories it contains.
+* -r (recursive) — Deletes a directory and all files and subdirectories it
+  contains.
+* Combining the flags `-rf` will do a recursive force delete.
+
+You can also delete directories using `rm` or `rmdir`. If you want to remove all
+folders and files in a directory the dull command is `rm -rf name_of_directory`.
 
-You can also delete directories using `rm` or `rmdir`.
+**Note:** When using `rm -rf`, be _very_ careful. If you submit a command like
+`rm -rf /` , if you will delete the root directory!
 
 ## Conclusion
 
-As we keep exploring and working with the command line, we will start to unlock and
-understand its full potential! Adopting the terminal for command-related interactions
-can allow us to become more productive users-—we work on multiple projects, tasks, and
-easily switch contexts and folders. It's also the root of all computing systems. While
-syntax might vary between operating systems, the functions that we are given through
-terminal applications are the same.
+With these commands, we can begin to organize our file system in a manageable
+way. We illustrated how to create, move, rename, copy, and delete files and
+directories as well as how to avoid causing some chaos with the naming and
+removing of files and directories. Most commands also have many useful flags
+that can give us additional controls and options. As we get accustomed to
+navigating and manipulating file structures in the command line, we will start
+to grasp the power of `bash`.
```

Comment:
With just a little bit of practice you'll probably find yourself putting Finder or Windows Explorer away and reaching for the CLI. Welcome to the club!